### ISSUE TITLE
Running docker container ends with "ERROR    argument `-c/--client-count`: invalid with: '26'. Exiting..."

### ISSUE THREAD

**Issue by spitfire** [SystemRage/py-kms#80]
Running docker container with `docker run --name py3-kms --restart=unless-stopped    -p 8080:8080     -p 1688:1688     -e IP=0.0.0.0     -e PORT=1688     -e SQLITE=true     -e HWID=RANDOM     -e LOGLEVEL=INFO     -e LOGSIZE=2     -e LOGFILE=/var/log/py3-kms.log     -v /etc/localtime:/etc/localtime:ro     -v /var/log:/var/log:rw     --restart=unless-stopped pykmsorg/py-kms`ends with "ERROR    argument `-c/--client-count`: invalid with: '26'. Exiting..." all over the log and the container in a restart loop

**Comment by Simonmicro** [comment]
Duplicate of #75

**Comment by Dreikasehoch** [comment]
It looks like there hasn't been a rebuild of the official docker containers in nearly 2 weeks.
https://hub.docker.com/r/pykmsorg/py-kms/builds
In the meantime, a quick `docker build` could get you up and running.

**Comment by Simonmicro** [comment]
I'm sorry, but it seems the docker infrastructure is having problems https://forums.docker.com/t/build-stuck-in-pending-on-docker-hub/1360/15
The build which fixes this are stuck since 4 days now... Sorry...

**Comment by spitfire** [comment]
Thanks for the info @Simonmicro! I'm not in that much of a hurry (and I don't really want to build the container), can you let us know (here) once this is fixed on Docker hub's side and a new build is ready.

**Comment by Simonmicro** [comment]
@spitfire I can try, but the problem has been there for a while now (god damn you docker). We'll have to see.
Just in case I miss the rebuild: Try using watchtower - so you'll get future updates too.

Stay tuned, Simon

**Comment by Simonmicro** [comment]
Okay, the docker container is now rebuild... And still broken due the count error #75. I'll fix this now.

**Comment by Simonmicro** [comment]
@spitfire  Fixed. Please close this issue now.
